completor/utils.py

```diff
@@ -338,7 +338,7 @@ def find_well_keyword_data(well: str, keyword: str, text: str) -> str:
                 if once:
                     lines.append(line)
                 continue
-            if well in line.split()[0]:
+            if well == line.split()[0].strip("'\""):
```

Comment:
I read the original meaning here to be removal of both ' and " symbols, but this does not work as intended if my assumption is correct, so I suggest something like this:
PS: Added an example below
```suggestion
            if well == line.split()[0].replace("'", "").replace('"', ""):
```
```python
>>> print(' AYy " '.strip("'\""))
 AYy " 
>>> print(' AYy " '.replace("'", "").replace('"',""))
 AYy  
```